tutorials/ner_multiple_trials/scripts/evaluate_with_trials.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+from typing import Optional, Iterable
+from enum import Enum
+
+import typer
+from spacy.cli._util import Arg, Opt, import_code
```

Comment:
To avoid relying on (private) internals too much, I wouldn't import `Arg` and `Opt` from `_util`

Reply:
Updated in https://github.com/explosion/projects/pull/123/commits/16d7d1371d8df64eadbf8f752091833d0a13734f